pages/htmlresources.md

```diff
@@ -76,15 +76,24 @@ pip install http-parser
 ```
 
 ### Generating the couchapp
-Using `couchapp generate` command you will be able to generate the structure of a couchapp, but first, you need to create a folder with the app name and then, access to it.
+Now you should be in the _couchapp_ folder. Using `couchapp generate` command you will be able to generate the structure of a couchapp, but first, you need to create a folder with the app name and then, access to it.
```

Comment:
Helping fixing grammar will improve clarity more then just adding a small step. Try changing it to something like this:

"When using `couchapp generate` command you will be able to generate the structure of a basic couchapp. First, make sure you are in _couchapp_ folder. Then create a folder with the same app name you are making. Finally, change to that folder you just created. "